src/com/linkedin/parseq/Task.java

```diff
@@ -64,6 +64,8 @@
 public interface Task<T> extends Promise<T>, Cancellable {
   static final Logger LOGGER = LoggerFactory.getLogger(Task.class);
 
+  static final TaskDescriptor _taskDescriptor = TaskDescriptorFactory.getTaskDescriptor();
```

Comment:
Some usages were nor replaced e.g. line 399 (before changes):
```
  /**
   * Equivalent to {@code andThen("andThen", consumer)}.
   * @see #andThen(String, Consumer1)
   */
  default Task<T> andThen(final Consumer1<? super T> consumer) {
    return andThen("andThen: " + consumer.getClass().getName(), consumer);
  }
```

Can you verify other usages?